dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -9,11 +9,10 @@ ENV DEBIAN_FRONTEND="noninteractive"
 RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
         m4 automake autoconf time wget rsync git gcc-multilib opam \
         libgtk2.0-dev libgtksourceview2.0-dev \
-        texlive-latex-extra texlive-fonts-recommended texlive-science tipa \
-        python3-sphinx python3-pexpect python3-sphinx-rtd-theme python3-bs4 python3-sphinxcontrib.bibtex \
-        python3-setuptools python3-wheel python3-pip
+        texlive-full python3-pip python3-setuptools
```

Comment:
`texlive-full` seems also overkill here;  note that the size of the Docker image is an important variable.

Reply:
OK I see the problem. Compared to the packages that were previously there, one additional dependency is `xelatex`.

Reply:
As I was fearing image size has increased by almost 1GiB (https://gitlab.com/coq/coq/container_registry) IMO we cannot take that.

This PR seems a bit too large BTW, wouldn't it make sense to start integrating some of the features?

Reply:
texlive-full depends on various heavy doc packages which we should be able to avoid.
eg on my system there is `texlive-latex-extra-doc` at 534MB uncompressed.
This is debian so maybe ubuntu does it differently but I doubt that.

BTW maybe we should `apt-get clean` to get rid of the downloaded .deb files after we're done installing.

Reply:
> This PR seems a bit too large BTW, wouldn't it make sense to start integrating some of the features?

I'm fine with that :) But what features are you thinking of?